sdk/cosmos/azure-cosmos-spark_3-0_2-12/docs/configuration-reference.md

```diff
@@ -0,0 +1,59 @@
+Configuration Reference:
+
+
+## Generic Configuration
+
+
+| Config Property Name      | Default | Description |
+| :---        |    :----   |         :--- | 
+| `spark.cosmos.accountEndpoint`      | None   | Cosmos DB Account Endpoint Uri |
+| `spark.cosmos.accountKey`      | None    | Cosmos DB Account Key  |
+| `spark.cosmos.database`      | None    | Cosmos DB database name  |
+| `spark.cosmos.container`      | None    | Cosmos DB container name  |
+
+
+### Additional tuning
+
+
+
+| Config Property Name      | Default | Description |
+| :---        |    :----   |         :--- | 
+| `spark.cosmos.useGatewayMode`      | `false`    | Use gateway mode for the client operations  |
+| `spark.cosmos.read.forceEventualConsistency`      | `true`    | Makes the client use Eventual consistency for read operations  |
+| `spark.cosmos.applicationName`      | None    | Application name  |
```

Comment:
instead of using the default account level consistency

Reply:
Thanks. DONE